DependencyInjection/Compiler/DriverCompilerPass.php

```diff
@@ -23,6 +23,8 @@ class DriverCompilerPass extends AbstractCompilerPass
 {
     /**
      * {@inheritdoc}
```

Comment:
imho we should remove these empty phpdoc everywhere (could be a separate MR), as they do not add information.